internal/agent/client/management.go

```diff
@@ -79,3 +101,19 @@ func (m *management) GetRenderedDevice(ctx context.Context, name string, params
 
 	return nil, resp.StatusCode(), nil
 }
+
+func PatchDeviceStatusSystemInfo(ctx context.Context, client Management, name string, systemInfo v1alpha1.DeviceSystemInfo) error {
```

Comment:
Isn't this some leftover ? I dont see it being used anywhere

Reply:
future implementation but to your point not needed for the PR